### DISCUSSION TITLE
Jupyter link and display join condition on the arrow connections

### DISCUSSION THREAD

**Discussion by jonnywebcash** [reata/sqllineage#588] [Q&A]
Hi, I'm an enthusiastic new user of SQLLineage in Python. Thanks to everybody for the superb work done!
I have two questions:

I tried to run the code on Jupyter Lab, but when I opened the host link, it didn't work.
Is it possible to display the 'filter' or the condition (for example, join condition) in the lineage column on the arrow connecting the elements of the table?
Thanks to everybody for the help!"

**Comment by reata** [comment]
The thing we can add to column lineage is column transformation logic. join or where clause does not fit well in column lineage modeling. We might expose them via other API in future release. But that's not on short term plan.
